### PR TITLE
NTR protoplasmic astrocytes fibrous astrocytes interlaminar astrocytes

### LINKED ISSUE #2320: [NTR-cxg] protoplasmic astrocyte
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl

**Preferred term label**
protoplasmic astrocyte

**Synonym(s)** (add reference(s), please)
[Astro L1-6 FGFR3 SLC14A1 middle temporal gyrus astrocyte (Hsap) (PCL:0023071)](http://purl.obolibrary.org/obo/PCL_0023071)
[Astro L1-6 FGFR3 PLCG1 primary motor cortex astrocyte (Hsap) (PCL:0015124)](http://purl.obolibrary.org/obo/PCL_0015124)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
An astrocyte of the cerebral cortex that is found in all cortical layers and exhibits a highly branched morphology.
PMID: 19279265, 31435019, 34616062

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
astrocyte (CL:0000127)

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
cerebral cortex (UBERON:0000956)

**Your ORCID**
0000-0001-5287-0284

**Additional notes or concerns** If there are markers that uniquely identify the cell type in a particular species, they can be added here.

### PR DESCRIPTION
**Pull request by AvolaAmg**
Fixes #2320 #2321 #2322 #2328

### CODE REVIEW

**Comment by dosumis** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32019,6 +32024,50 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
...
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042007 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042007 "2024-04-08T16:01:24Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:20012068") oboInOwl:hasDbXref obo:CL_4042007 "This astrocyte extends its branches to several hundred of dendrites to multiple neurons to envelope 100.000 or more synapses. In humans, a protoplasmic astrocyte has an average diameter of 142 µm, and its diameter ranges between 100 and 400 µm. In rodents the average diameter of a protoplasmic astrocites is 56 µm.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32032]: 100.000 --> 100,000 ?

Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32032]: Sorry European spelling, you are right!

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32019,6 +32024,50 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
...
+AnnotationAssertion(rdfs:label obo:CL_4042007 "protoplasmic astrocyte"@en)
+SubClassOf(obo:CL_4042007 obo:CL_0002605)
+
+# Class: obo:CL_4042008 (fibrous astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:22144298") Annotation(oboInOwl:hasDbXref "PMID:28280934") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:34616062") obo:IAO_0000115 obo:CL_4042008 "An astrocyte cell with long fiber-like processes, the soma of this astrocyte is part of a white matter, its soma and processes are oriented along the white matter tracts. This astrocyte type has several contacts with the vasculature of a nervous system and is involved in metabolic support.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32038]: Minor quibble with the way  this is phrased:

> An astrocyte cell with long fiber-like processes, the soma of this astrocyte is part of a white matter, its soma and processes are oriented along the white matter tracts

--> 

An astrocyte cell with long fiber-like processes with a soma in the white matter  and processes are oriented along the white matter tracts part of a white matter.  

This better matches the 'genus-differentia pattern' recommended for defs.

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -32019,6 +32024,50 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
...
+
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042009 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042009 "2024-04-08T16:33:45Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30552685") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042009 "ILA")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:30552685") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:32930323") Annotation(oboInOwl:hasDbXref "PMID:34616062") Annotation(oboInOwl:hasDbXref "PMID:37824655") rdfs:comment obo:CL_4042009 "An astrocyte type that presents radial protrusions across the layers of a cortex. The soma of this astrocyte is part of the first layer of a neocortex. This astrocyte extents its protrusions transversally to the deeper layers of a cortex and it creates contact with neurons, a pia matter and capillaries. This astrocyte is involved in facilitating the communication across neurons, astrocytes, capillaries, meninges and the cerebrospinal fluid.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32051]: > a pia matter
-->
the pia mater

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -32019,6 +32024,50 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
...
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042007 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042007 "2024-04-08T16:01:24Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:20012068") rdfs:comment obo:CL_4042007 "This astrocyte extends its branches to several hundred of dendrites to multiple neurons to envelope 100,000 or more synapses. In humans, a protoplasmic astrocyte has an average diameter of 142 µm, and its diameter ranges between 100 and 400 µm. In rodents the average diameter of a protoplasmic astrocites is 56 µm.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32032]: Also wrong AP? Text showing up as on dbxref rather than comment in diff.

**Review by dosumis — CHANGES_REQUESTED** [review]
Please add part relationships (e.g. to cortex).  Should be able to get some inferred classification from these.  (Note: This looks like a case where has_soma_location would be useful, but this is currently restricted to neurons).

Besides that- please see minor correction/suggestions to text

(I'm also seeing something odd with APs in robot diffs.  Investigating)

File: src/ontology/cl-edit.owl (hunk 4 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32032]: It is fixed now

**Comment by dosumis** [comment]
@anitacaron  - is there something odd going on with the  ROBOT diff?  I'm seeing different APs on ROBOT diff vs functional syntax text diff

e.g. 

[CL_4042009](http://purl.obolibrary.org/obo/CL_4042009) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "An astrocyte type that presents radial protrusions across the layers of a cortex. The soma of this astrocyte is part of the first layer of a neocortex. This astrocyte extents its protrusions transversally to the deeper layers of a cortex and it creates contact with neurons, a pia matter and capillaries. This astrocyte is involved in facilitating the communication across neurons, astrocytes, capillaries, meninges and the cerebrospinal fluid."

vs: 

AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:22144298") Annotation(oboInOwl:hasDbXref "PMID:28280934") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:34616062") obo:IAO_0000115 obo:CL_4042008 "An astrocyte cell with long fiber-like processes with a soma in the white matter and processes are oriented along the white matter tracts part of a white matter. This astrocyte type has several contacts with the vasculature of a nervous system and is involved in metabolic support.")

IAO_0000115 = definition


Looks correct on loading in Protege.

image

**Comment by dosumis** [comment]
Oops - pasto in last comment. Correcting.

**Comment by AvolaAmg** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 5]
```diff
@@ -32019,6 +32024,55 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32930323") Annotation(oboInOwl:hasDbXref "PMID:37139179") obo:IAO_0000115 obo:CL_4042010 "An interlaminar astrocyte whose soma is part of the first layer of a neocortex and is in contact with a pia surface.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042010 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042010 "2024-04-08T16:37:57Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4042010 "pial interlaminar astrocyte"@en)
+SubClassOf(obo:CL_4042010 obo:CL_4042009)
+
+# Class: obo:CL_4042011 (subpial Interlaminar astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32930323") Annotation(oboInOwl:hasDbXref "PMID:37139179") obo:IAO_0000115 obo:CL_4042011 "An interlaminar astrocyte type whose soma is part of the upper first layer of the neocortex and its processes extend to a pia surface.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32070]: Consider adding 'connected to' some pia mater'

**Comment by dosumis** [comment]
@JABelfiore can you review text on this PR?  Given your Comms background I'm hoping you might be able to help polish/subedit definitions and comments.

File: src/ontology/cl-edit.owl [hunk 6]
```diff
@@ -32019,6 +32024,56 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
 SubClassOf(obo:CL_4042006 obo:CL_4042003)
 SubClassOf(obo:CL_4042006 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002092))
 
+# Class: obo:CL_4042007 (protoplasmic astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:25904839") Annotation(oboInOwl:hasDbXref "PMID:34616062") Annotation(oboInOwl:hasDbXref "PMID:37139179") Annotation(oboInOwl:hasDbXref "PMID:37824655") obo:IAO_0000115 obo:CL_4042007 "An astrocyte cell that has highly branched protrusions, its soma is part of a neocortex and it is found between layer 2 and 6. A protoplasmic astrocyte is involved in the formation and elimination of synapses, in the clearance of glutamate, in the modulation of synaptic functions and in the regulation of blood flow in response to synaptic activity.")
```
Comment (JABelfiore) [inline src/ontology/cl-edit.owl:32029]: Suggest: " A cell type with highly branched protrusions, found in neocortex layers 2-6, typically involved with the formation and elimination of synapses, glutamate clearance, modulation of synaptic functions and regulation of blood flow in response to synaptic activity."

File: src/ontology/cl-edit.owl [hunk 7]
```diff
@@ -32019,6 +32024,56 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
...
+AnnotationAssertion(rdfs:label obo:CL_4042007 "protoplasmic astrocyte"@en)
+SubClassOf(obo:CL_4042007 obo:CL_0002605)
+SubClassOf(obo:CL_4042007 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001950))
+SubClassOf(obo:CL_4042007 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001950))
+
+# Class: obo:CL_4042008 (fibrous astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:22144298") Annotation(oboInOwl:hasDbXref "PMID:28280934") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:34616062") obo:IAO_0000115 obo:CL_4042008 "An astrocyte cell with long fiber-like processes with a soma in the white matter and processes are oriented along the white matter tracts part of a white matter. This astrocyte type has several contacts with the vasculature of a nervous system and is involved in metabolic support.")
```
Comment (JABelfiore) [inline src/ontology/cl-edit.owl:32040]: Suggest "A cell type located in the white matter of the cerebral cortex that provides structural and metabolic support through long fibre-like processes that extend along the white matter tract and remain in close contact with the vasculature of the nervous system."

File: src/ontology/cl-edit.owl [hunk 8]
```diff
@@ -32019,6 +32024,56 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
...
+AnnotationAssertion(rdfs:label obo:CL_4042008 "fibrous astrocyte"@en)
+SubClassOf(obo:CL_4042008 obo:CL_0000127)
+SubClassOf(obo:CL_4042008 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002316))
+SubClassOf(obo:CL_4042008 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002316))
+
+# Class: obo:CL_4042009 (interlaminar astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:30552685") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:32930323") Annotation(oboInOwl:hasDbXref "PMID:34616062") Annotation(oboInOwl:hasDbXref "PMID:37824655") obo:IAO_0000115 obo:CL_4042009 "An astrocyte type that presents radial protrusions across the layers of a cortex. The soma of this astrocyte is part of the first layer of a neocortex. This astrocyte extents its protrusions transversally to the deeper layers of a cortex and it creates contact with neurons, the pia matter and capillaries. This astrocyte is involved in facilitating the communication across neurons, astrocytes, capillaries, meninges and the cerebrospinal fluid.")
```
Comment (JABelfiore) [inline src/ontology/cl-edit.owl:32051]: Suggest: "A cell type located in the first layer of the neocortex with radial protrusions extending transversely into the deeper cortex layers, herby facilitating communication across neurons, astrocytes, capillaries, meninges and cerebrospinal fluid through contact with neurons, pia mater and capillaries."

Comment (dosumis) [reply src/ontology/cl-edit.owl:32051]: "An astrocyte located in 

A ~~cell type~~ _astrocyte_ located in the first layer of the neocortex with radial protrusions extending transversely into the deeper cortex layers, ~~herby~~ ..

ALso consider moving structural information - contact with pia mater and capillaries closer to start

**Review by dosumis — DISMISSED** [review]
Please make minor changes in text before merge.  Thanks!

**Review by JABelfiore — APPROVED** [review]

File: src/ontology/cl-edit.owl [hunk 9]
```diff
@@ -32022,6 +32027,56 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
...
+AnnotationAssertion(rdfs:label obo:CL_4042007 "protoplasmic astrocyte"@en)
+SubClassOf(obo:CL_4042007 obo:CL_0002605)
+SubClassOf(obo:CL_4042007 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001950))
+SubClassOf(obo:CL_4042007 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001950))
+
+# Class: obo:CL_4042008 (fibrous astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:22144298") Annotation(oboInOwl:hasDbXref "PMID:28280934") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:34616062") obo:IAO_0000115 obo:CL_4042008 "A cell type located in the first layer of the neocortex with radial protrusions extending transversely into the deeper cortex layers, herby facilitating communication across neurons, astrocytes, capillaries, meninges and cerebrospinal fluid through contact with neurons, pia mater and capillaries.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32043]: I would suggest to change "A cell type..." to "An astrocyte..."

**Pull request merged**